<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="addWidget_button">
      <property name="text">
       <string>Сгенерировать новость</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Пауза</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="slider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="openStatistic_button">
      <property name="text">
       <string>Статистика пользователей</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="styleSheet">
       <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="format">
       <string>%vx</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QProgressBar" name="usersCountBar">
      <property name="styleSheet">
       <string notr="true">font: 900 9pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="format">
       <string>%v</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QSlider" name="horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="generateUsers_button">
      <property name="text">
       <string>Сгенерировать пользователей</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>988</width>
         <height>337</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="vlayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>usersCountBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
